src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertySymbolBase.cs

```diff
@@ -783,6 +790,15 @@ internal bool IsAutoProperty
                 return null;
             }
         }
+
+        private bool AllowsInitializer
```

Comment:
`HasNonErrorBackingField` might be a better name that's also suitable for the usage in `AllowedAttributeLocations`. Let me know what you think.